uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget-absence-types-available-balance.component.js

```diff
@@ -50,7 +51,19 @@ define([
      * @return {Boolean}
      */
     function areBindingsReady () {
-      return vm.absenceTypes && vm.absencePeriod && vm.contactId;
+      return vm.absenceTypes && vm.absencePeriod && vm.contactId &&
+        vm.jobContract;
+    }
+
+    /**
+     * Returns a list of IDs of absence types the contact has entitlements for
+     *
+     * @return {Array}
+     */
+    function getContractAbsenceEntitlements () {
```

Comment:
The name of the func sounds like it loads entitlements. The word 'absence' is also not really needed I think. Could we rename in let's say `getContractEntitlementsIds`?